tfx/components/trainer/fn_args_utils_test.py

```diff
@@ -69,10 +69,10 @@ def testGetCommonFnArgs(self):
     self.assertEqual(fn_args.eval_steps, 500)
     self.assertLen(fn_args.train_files, 1)
     self.assertEqual(fn_args.train_files[0],
-                     os.path.join(examples.uri, 'train', '*'))
+                     os.path.join(examples.uri, 'train', "*"))
```

Comment:
nit: were the quotes changes supposed to be included?

Reply:
@sngahane good nit. I got lazy with using `black`

Reply:
Still missing the single quotes changes here ;)

Reply:
ah, @sngahane I put it into a different commit: https://github.com/yanyuchka/tfx/pull/1/commits/08aa8e258bbfbdd9ec0c9ef83b5ac3f827c39775 :smile: